src/config/index.ts

```diff
@@ -43,7 +43,7 @@ const isNetworkId = (id: unknown): id is ETHEREUM_NETWORK => {
   return Object.values(ETHEREUM_NETWORK).some((network) => network === id)
 }
 
-export const NETWORK_ID_KEY = 'SAFE__networkId'
+export const NETWORK_ID_KEY = 'networkId'
```

Comment:
In the chains migration, I am trying to move all `networkId` references to `chainId`. Do you think it's worthwhile including this here too?

Reply:
Out of scope, I'm just removing the prefix here.